app/subscriber/src/components/date-filter/DateFilter.tsx

```diff
@@ -62,6 +62,7 @@ export const DateFilter: React.FC<IDateFilterProps> = ({ filter, storeFilter })
         <FaCalendarDay className="calendar" onClick={() => setOpen(true)} />
         <ReactDatePicker
           open={open}
+          maxDate={new Date()}
```

Comment:
Won't this always default today?  We don't want a value in the component if the user didn't select one.

Reply:
@Fosol yes exactly, it just sets a limit on the furthest date into the future that the user can select, it doesn't actually set any values... everywhere we use DateFilter currently this logic is valid

Reply:
Just to elaborate, these are the places that will have the calendar component with a max selectable date as today. I think they're all valid, are any of these pages areas you would need to pick a date in the future (beyond today)?

<img width="309" alt="Screenshot 2024-02-02 at 6 25 04 PM" src="https://github.com/bcgov/tno/assets/7937856/d06c46ab-ca16-4ef3-9f3d-41fa55c3669d">

If we need to add a prop to pass into `DateFilter` component, I can definitely do that, but right now everywhere would use the same maxDate, so it's not necessary yet. If we need to customize that, we can add the prop later